src/overlays/actors/ovl_Bg_Hidan_Fslift/z_bg_hidan_fslift.c

```diff
@@ -53,10 +53,10 @@ void BgHidanFslift_Init(Actor* thisx, PlayState* play) {
         Actor_Kill(&this->dyna.actor);
         return;
     }
-    this->actionFunc = func_80886FCC;
+    this->actionFunc = BgHidanFslift_WaitForPlayerOnTop;
 }
 
-void func_80886F24(BgHidanFslift* this) {
+void BgHidanFslift_CopyPosToHsblock(BgHidanFslift* this) {
     if (this->dyna.actor.child != NULL && this->dyna.actor.child->update != NULL) {
```

Comment:
I think `_SetHookshotTargetPos` could be good
Think its fine to refer to a more "normal" name to the actor instead of its internal name. 
With "copy pos to" it took me a second to figure out what direction the copy was happening. Plus, its not an exact copy since it does move it up a lil bit